### ISSUE TITLE
Document SECONDARY_ARCHITECTURES

### ISSUE THREAD

**Issue by alaviss** [haikuports/haikuports#1854]
Currently `SECONDARY_ARCHITECTURES` and it's related variable `secondaryArchSuffix` usage is not documented in [HaikuPorter BuildRecipes](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes)

**Comment by Begasus** [comment]
Maybe also related, document $commandSuffix, $commandBinDir, $libVersion and $libVersionCompat? :)

**Comment by pulkomandy** [comment]
These are not variables used by HaikuPorter, only a convention in recipes.

**Comment by Begasus** [comment]
Indeed, but if they are commonly used maybe some documentation about them would be handy? (at least for new comers) :)

**Comment by Begasus** [comment]
Just a quick one, if not relevant you can always remove it :)

[variables.txt](https://github.com/haikuports/haikuports/files/1579253/variables.txt)

**Comment by fbrosson** [comment]
I'd say that we should _not_ put them in the main doc.
We might, however, (and if someone finds the time to do that) add it to a different page, with frequently used tricks like the commandSuffix+commandBinDir, the suggested method to remove docs for secondary arch builds, etc...

IMO, contributors should be able to distinguish what is official and what is only used but not required.

**Comment by Begasus** [comment]
Correct there @fbrosson they are not required, but if they are commonly used it would be a good idea to put them "somewhere" ;)

**Comment by Begasus** [comment]
PS, for commandSuffix and commandBinDir ... sometimes they are "required"?

**Comment by fbrosson** [comment]
> PS, for commandSuffix and commandBinDir ... sometimes they are "required"?

These variables are _not_ required. What is required (well, prefered, to be exact) is the functionality we obtain with them. (We could choose any other names, but we like to stick to those.)
The best example is whena recipe does not build on x86_gcc2 primary arch: we use the trick to allow that recipe's `cmd:` items to be available without the `-x86` suffix.

**Comment by fbrosson** [comment]
> Maybe also related, document variables commandSuffix, commandBinDir, libVersion and libVersionCompat? :)

[Done](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines/_compare/8c3448b40fa433bd6372311ff156ea0772d4dfc7...94e45c96c7c422b73ae4f8baceb5dfec36433f17) :)

(BTW, it was easier to document them on the same page, after all.)
